### PR TITLE
add skipLogcatCapture

### PR DESCRIPTION
**Pull request by KazuCocoa**
Add ability to disable logcat capture 
https://github.com/appium/appium/issues/12203

cc @sshock

### CODE REVIEW

**Comment by sshock** [comment]
great, thanks for adding this!

**Comment by mykola-mokhnach** [comment]
@KazuCocoa Please add the new cap into the documentation

File: lib/driver.js [hunk 1]
```diff
@@ -416,7 +416,9 @@ class AndroidDriver extends BaseDriver {
     }
     // some cleanup we want to do regardless, in case we are shutting down
     // mid-startup
-    await this.adb.stopLogcat();
+    if (!this.opts.skipLogcatCapture) {
```
Comment (mykola-mokhnach) [inline lib/driver.js]: I think it's safe not to check this, since `stopLogcat` simply returns silently if no logs collections has been started

Comment (KazuCocoa) [reply lib/driver.js]: Okay, right. Will simplify here
https://github.com/appium/appium-adb/blob/84d736c6b64a66721b634e69494f8ffaa47cba81/lib/tools/adb-commands.js#L1119-L1128

**Pull request merged**